### DISCUSSION TITLE
Please help I did all the steps (macOS 13.7.2)

### DISCUSSION THREAD

**Discussion by Casondoesstuuffz** [ThioJoe/YT-Spammer-Purge#1146] [Q&A] [answered]
![Screenshot 2024-12-29 at 4 11 33 PM](https://github.com/user-attachments/assets/8ffe8fbc-ca75-4ea7-9f9c-a4a66cf552f4)

**Comment by Firecul** [comment]
It's possible the regex module is named something different on mac os so the install wouldn't be complete?

**Reply by Firecul** [reply]
you could try adding 
```
brew install regex && \
```
**OR**
```
brew install regex-opt && \
```
above this line 
https://github.com/ThioJoe/YT-Spammer-Purge/blob/91816f312807454b74a93abd3d6849cf4af3fac5/install.sh#L107
and then reinstalling?

The second one I have just found with a quick search so might help you? I have not read more than that so I can't promise anything.

**Reply by Casondoesstuuffz** [reply, marked as answer]
I asked chatgpt and it turns out some inports where missing and I had to download 1 or 2 things

**Reply by Firecul** [reply]
What were the things? Will be handy for the next person who has this issue

**Reply by Casondoesstuuffz** [reply]
I had to do more then one chats because im using the free plan let me see if I can find them

**Reply by Casondoesstuuffz** [reply]
> What were the things? Will be handy for the next person who has this issue

ok, I just re-asked chatgpt it heres the link https://chatgpt.com/share/6779d957-dff4-800a-978f-d2464d66b41c

**Reply by Firecul** [reply]
`pip3 install regex` then, fair enough.
